bitnami/elasticsearch/templates/ingest/statefulset.yaml

```
@@ -83,9 +83,9 @@ spec:
       {{- if .Values.ingest.terminationGracePeriodSeconds }}
       terminationGracePeriodSeconds: {{ .Values.ingest.terminationGracePeriodSeconds }}
       {{- end }}
-      {{- if or (or .Values.initContainers .Values.ingest.initContainers) .Values.security.enabled .Values.sysctlImage.enabled }}
+      {{- if and (default true .Values.enableDefaultInitContainers) (or (or .Values.initContainers .Values.ingest.initContainers) .Values.security.enabled .Values.sysctlImage.enabled) }}
```

If I am not wrong, we should set the initContainers if users set any of these values:
* `initContainers`
* `ingest.initContainers`
* `enableDefaultInitContainers` and `security.enabled ` 
* `enableDefaultInitContainers` and `sysctlImage.enabled`

So the condition should be:

```suggestion
      {{- if or .Values.initContainers .Values.ingest.initContainers (and  .Values.enableDefaultInitContainers (or .Values.security.enabled .Values.sysctlImage.enabled)) }}
```